frontend/src/modules/entitieslist/components/Entity.test.js

```diff
@@ -148,4 +148,54 @@ describe('<Entity>', () => {
         wrapper.find('li').simulate('click');
         expect(selectEntityFn.calledOnce).toEqual(true);
     });
+
+    it('does not call the selectEntity function on click on .status', () => {
+        const selectEntityFn = sinon.spy();
+        const wrapper = mount(<Entity
+            entity={ ENTITY_A }
+            selectEntity={ selectEntityFn }
+            locale={ DEFAULT_LOCALE }
+        />);
+        wrapper.find('.status').simulate('click');
+        expect(selectEntityFn.calledOnce).toEqual(false);
```

Comment:
If you want to check it's never been called, you need to test for `.called`. Otherwise, this will pass if the function is called twice or more.

Note that the same thing happens twice below.